templates/macvtap.yaml.in

```diff
@@ -24,7 +24,21 @@ spec:
         volumeMounts:
           - name: deviceplugin
             mountPath: /var/lib/kubelet/device-plugins
+        command: [ "/macvtap-deviceplugin", "-v", "3", "-logtostderr"]
+      initContainers:
+      - name: install-cni
+        image: {{ .ImageRegistry }}/{{ .ImageName }}:{{ .ImageTag }}
+        securityContext:
+          privileged: true
+        command: ['cp', '/macvtap-cni', '/host/opt/cni/bin/macvtap']
```

Comment:
Please make the destination into a variable. The CNI binary directory is different for OpenShift and Kubernetes.

Reply:
Should be similar to `{{ .ImageRegistry }}/{{ .ImageName }}:{{ .ImageTag }}`. We can replace it by a constant within this project. Once we include it in CNAO however, we would be replacing the value.

Reply:
Done.